src/imports.jl

```diff
@@ -83,7 +83,7 @@ function add_to_imported_modules(scope::Scope, name::Symbol, val)
 end
 no_modules_above(s::Scope) = !CSTParser.defines_module(s.expr) || s.parent === nothing || no_modules_above(s.parent)
 function get_named_toplevel_module(s::Scope, name::String)
-    if CSTParser.defines_module(s.expr) && valofid(CSTParser.get_name(s.expr)) == name && no_modules_above(s)
+    if CSTParser.defines_module(s.expr) && CSTParser.isidentifier((m_name = CSTParser.get_name(s.expr);)) && valofid(m_name) == name && no_modules_above(s)
```

Comment:
This reads really weird (even though it makes sense that it works). Would you mind making this two different `if` statements?